### PR TITLE
Add message.message to message tokenize hook deps array

### PR DESCRIPTION
**Pull request by AhyoungRyu**
> This change is originally made by @ishubham21 at https://github.com/sendbird/sendbird-uikit-react/pull/916. Please refer to this PR for more detailed background.

`message.updateAt` won't be updated if the message is coming from a bot especially in streaming. Its value gets just `0`. I added one more field `message.message` to the dependency array so we can make sure the hook is being executed again.

### CODE REVIEW

**Review by bang9 — APPROVED** [review]
lgtm

Do you know if this is the intended behavior on the server side or if it's a bug?

**Comment by AhyoungRyu** [comment]
Yeah we've been discussed multiple times and decided not to update this `updatedAt` field. 
If the field gets updated while the message is being sent via streaming, `(edited)` will be printed too. We could use handle this side effect separately, but actually, the streaming message doesn't mean the message gets "updated". 
Not sure it sounds clear 😅

File: src/modules/Thread/components/ParentMessageInfo/ParentMessageInfoItem.tsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ export default function ParentMessageInfoItem({
     return tokenizeMessage({
       messageText: (message as UserMessage)?.message,
     });
-  }, [message?.updatedAt]);
+  }, [message?.updatedAt, (message as UserMessage)?.message]);
```
Comment (liamcho) [inline src/modules/Thread/components/ParentMessageInfo/ParentMessageInfoItem.tsx:94]: is it always user message?

Comment (AhyoungRyu) [reply src/modules/Thread/components/ParentMessageInfo/ParentMessageInfoItem.tsx:94]: Probably not. I did this for typing. See 92 line too.

Comment (liamcho) [reply src/modules/Thread/components/ParentMessageInfo/ParentMessageInfoItem.tsx:94]: Can we please add comment so people don't get confused? same for all others below.

**Comment by liamcho** [comment]
Should we add a changelog for this "bug fix"?

**Comment by AhyoungRyu** [comment]
@liamcho Yes

**Pull request merged**